uber/utils.py

```diff
@@ -949,15 +949,27 @@ def charge_cc(self, session, token):
             return 'An unexpected problem occurred while processing your card: ' + str(e)
         else:
             if self.models:
-                session.add(self.stripe_transaction_from_charge())
+                txn = self.stripe_transaction_from_charge()
+                for model in self.models:
+                    if model.__class__.__name__ == "Attendee":
+                        session.add(uber.models.commerce.StripeTransactionAttendee(
+                            txn_id=txn.id,
+                            attendee_id=model.id,
+                            share=model.amount_unpaid * 100
+                        ))
+                    elif model.__class__.__name__ == "Group":
+                        session.add(uber.models.commerce.StripeTransactionGroup(
+                            txn_id=txn.id,
+                            group_id=model.id,
+                            share=model.amount_unpaid * 100
+                        ))
+                session.add(txn)
```

Comment:
So.

When I first wrote this up, I tried preregistering an attendee and got a "key is not present in table" error for the attendee ID/foreignkey that the StripeTransactionAttendee row was using. I added a `session.add(attendee)` to fix this. Later, I found that kicking in extra broke that (because the attendee already exists in the DB), so I changed it to `session.merge(attendee)`. While that fixed kicking in extra, I tried to preregister another attendee and it threw a "key already exists" error for the attendee (???). I finally took the line out entirely just to go back to square one. I got no more errors.

I'm suspicious, to say that least.